src/components_Todo/ErrorNotification.tsx

```diff
@@ -0,0 +1,47 @@
+import classNames from 'classnames';
+import { useEffect } from 'react';
+
+/* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
Delete this comment.

Try to add `aria-label` attribute to buttons all over the project to avoid using `eslint-disable`.
More information  [here]( https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/control-has-associated-label.md)